docs/SETUP_PROVISIONING.md

```diff
@@ -391,14 +391,72 @@ again in subsequent runs. ``ops/scripts/ansible_init.sh`` will also source that
 with a Docker daemon. In addition, a PostgreSQL database is created on the RDS
 instance using `sql/production_like.pgdmp`.
 
-You can install Ansible on macOS using [HomeBrew](https://brew.sh) with the commands
+Details of the Ansible changelog can be found at [here](https://docs.ansible.com/ansible/latest/reference_appendices/release_and_maintenance.html#ansible-community-changelogs).
+
+You can install the latest Ansible on macOS with the commands:
+```
+$ brew update
+$ brew install ansible
+$ ansible --version
+ansible [core 2.18.1]
+  config file = None
+  configured module search path = ['/Users/kencho/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
+  ansible python module location = /opt/homebrew/Cellar/ansible/11.1.0/libexec/lib/python3.13/site-packages/ansible
+  ansible collection location = /Users/kencho/.ansible/collections:/usr/share/ansible/collections
+  executable location = /opt/homebrew/bin/ansible
+  python version = 3.13.0 (main, Oct  7 2024, 05:02:14) [Clang 16.0.0 (clang-1600.0.26.4)] (/opt/homebrew/Cellar/ansible/11.1.0/libexec/bin/python)
+  jinja version = 3.1.4
+  libyaml = True
+```
+
+>**Note:** if you have installed the latest version, you can uninstall it with `brew uninstall ansible` and re-install the specific version as described below.
+>**Note:** but not any ansible version will be working, as you can see from the following issues reported by the internal developers who use mac with apple chip:
+
+| version                         | reported issues/warnings/descriptions                                                                                                                       | install this version    |
+|---------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------|
+| Ansible 11, ansible-core 2.18.1 | [WARNING]: Collection prometheus.prometheus does not support Ansible version 2.18.1, reported to https://github.com/prometheus-community/ansible/issues/490 | Yes                     |
+| Ansible 10, ansible-core 2.17+  | Not working because of python dependency                                                                                                                    | No                      | 
```

Comment:
I am using Ansible 10 (version 2.17) on home iMac. Have managed to successfully execute all playbooks including `files_playbook.yml`.